src/Hazelcast.Net.Tests/Serialization/Compact/SchemasLocalTests.cs

```diff
@@ -52,6 +53,11 @@ public async Task<ClientMessage> SendAsync(ClientMessage requestMessage, bool tr
             return await _respond(requestMessage);
         }
 
+        public Task<ClientMessage> SendToMemberAsync(ClientMessage message, MemberConnection memberConnection, CancellationToken cancellationToken = default)
+        {
+            throw new NotImplementedException();
```

Comment:
What about them? When will you complete?

Reply:
Will not - this is a stub messaging object, we only need to implement what's needed for the tests where it's used.